templates/rhdm71-kieserver.yaml

```diff
@@ -59,103 +59,120 @@ parameters:
+  name: KIE_SERVER_ROUTER_SERVICE
   required: false
-- displayName: KIE Server Bypass Auth User
-  description: KIE server bypass auth user. If this parameter is set to true, the Decision Server accepts
-   API calls without user account authorization. (Sets the org.kie.server.bypass.auth.user system property).
-  name: KIE_SERVER_BYPASS_AUTH_USER
-  value: 'false'
+- displayName: Smart Router Host
+  description: "The host name of the smart router, which could be the service name resolved by OpenShift or a globally resolvable domain name"
+  name: KIE_SERVER_ROUTER_HOST
+  example: "myapp-smartrouter"
+  required: false
+- displayName: Smart Router listening port
+  description: Port in which the smart router server listens (router property org.kie.server.router.port)
+  name: KIE_SERVER_ROUTER_PORT
+  example: "9000"
+  required: false
+- displayName: Smart Router protocol
+  description: KIE server router protocol (Used to build the org.kie.server.router.url.external property)
+  name: KIE_SERVER_ROUTER_PROTOCOL
+  example: "http"
   required: false
 - displayName: KIE Server Controller User
-  description: KIE server controller user name. The Decision Server uses this user account to log on to Decision 
-    Central or a controller. (Sets the org.kie.server.controller.user system property).
+  description: KIE server controller username (Sets the org.kie.server.controller.user system property)
   name: KIE_SERVER_CONTROLLER_USER
   value: controllerUser
   required: false
 - displayName: KIE Server Controller Password
-  description: KIE server controller password (sets the org.kie.server.controller.pwd system property).
+  description: KIE server controller password (Sets the org.kie.server.controller.pwd system property)
   name: KIE_SERVER_CONTROLLER_PWD
-  from: "[a-zA-Z]{6}[0-9]{1}!"
-  generate: expression
-  required: false
-- displayName: KIE server controller protocol
-  description: KIE server controller service. To use Decision Central that is hosted on the same OpenShift environment
-      to manage this Decision Server, set this variable to the OpenShift service name for the Decision Central. In this
-      case, do not set the KIE server controller protocol, KIE server controller host, and KIE server controller port.
-      (Sets the org.kie.server.controller system property if host and port are not set).
-  name: KIE_SERVER_CONTROLLER_PROTOCOL
-  value: http
   required: false
-- displayName: KIE server controller service
-  description: The service name for the optional Decision Central or controller, where it can be reached and registered with. (Used to set the org.kie.server.controller system property if host and port aren't set)
+- displayName: KIE Server Controller Service
+  description: The service name for the optional business-central-monitor, where it can be reached and registered with, to allow monitoring console functionality
```

Comment:
Decision Central or controller instead of BC monitoring

Reply:
Fixed.